app/controllers/comments_controller.rb

```diff
@@ -5,7 +5,14 @@ def show
 
   def create
     @post = Post.find(params[:id])
-    @comment = @post.comments.create(comment_params)
+    @comment = @post.comments.new(comment_params)
+    flash[:alert] = if @comment.save
+                      'Comment was created successfully'
+                    # redirect_to user_post_path(user_id: params[:user_id], id: @post.id)
+                    else
+                      'Could not save the comment'
+                      # redirect_to user_posts_path(user_id: params[:user_id], id: @post.id)
```

Comment:
- [ ] [OPTIONAL] Please remove the commented code to improve the quality of your code